Examples/Detectors/DD4hepDetector/src/GeometryIdentifierHooks.cpp

```diff
@@ -0,0 +1,29 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "ActsExamples/DD4hepDetector/GeometryIdentifierHooks.hpp"
+
+namespace det {
+namespace GeometryIdentifierHooks {
+
+Acts::GeometryIdentifier stripEndcapODD(Acts::GeometryIdentifier identifier,
+                                        const Acts::Surface& surface) {
+  if (identifier.volume() == 28 || identifier.volume() == 30) {
```

Comment:
A bit dangerous if the volume numbering changes, can't you change that to a callable struct and then make it such that you can add these parameters at struct construction (including the radius below).

Reply:
I can try to write some thing more generic. The issue is how to make it work with the bindings but maybe I can transform this into a function that take the volumes and radius as parameters and that return an std::function. I will have a try